.github/workflows/sync-supported-chains.yml

```diff
@@ -227,7 +227,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Trigger notification action on test repo
-        if: github.event_name == 'push' && (startsWith(github.ref, 'refs/heads/master') || startsWith(github.ref, 'refs/heads/release/'))
+        if: github.event.workflow_run.head_branch == 'master' || contains(github.event.workflow_run.head_branch, 'release/')
```

Comment:
not sure about that `contains`, shouldn't be `startswith`?

Reply:
Valid point - but hard to verify how branch is provided with workflow_run so will verify shortly after merge if all is ok.

"push" prfixes with "refs/heads" where workflow_run should not but GH never stops surprising me. That's whY i chosen contains at first place but may hit some edge case with weird branches like "test/release/1.28.0" (which can happen)